backend/src/model/dto/plantings.rs

```diff
@@ -0,0 +1,114 @@
+//! All DTOs associated with [`PlantingDto`].
```

Comment:
Why exactly do plantings entities and DTOs need to be placed in their own module?

Reply:
The DTOs file is already really large so it is much easier to read and change the code when its split like this. In my opinion we should actually split up the whole DTOs and entities file and move the implementations to the structs themselves. Ideally move the db implementation to a separate module. They should be part of model.